scripts/exregional_refl2tten.sh

```diff
@@ -0,0 +1,297 @@
+  ;;
+#
+"JET")
+  ulimit -s unlimited
+  ulimit -a
+  APRUN="srun"
+#  LD_LIBRARY_PATH="${UFS_WTHR_MDL_DIR}/FV3/ccpp/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+  ;;
+#
+"ODIN")
+#
+  module list
+
+  ulimit -s unlimited
+  ulimit -a
+  APRUN="srun -n ${PE_MEMBER01}"
+  ;;
+#
+esac
+#
+#-----------------------------------------------------------------------
+#
+# Extract from CDATE the starting year, month, day, and hour of the
+# forecast.  These are needed below for various operations.
+#
+#-----------------------------------------------------------------------
+#
+set -x
+START_DATE=`echo "${CDATE}" | sed 's/\([[:digit:]]\{2\}\)$/ \1/'`
+YYYYMMDDHH=`date +%Y%m%d%H -d "${START_DATE}"`
+JJJ=`date +%j -d "${START_DATE}"`
+
+YYYY=${YYYYMMDDHH:0:4}
+MM=${YYYYMMDDHH:4:2}
+DD=${YYYYMMDDHH:6:2}
+HH=${YYYYMMDDHH:8:2}
+YYYYMMDD=${YYYYMMDDHH:0:8}
+#
+#-----------------------------------------------------------------------
+#
+# Create links in the subdirectory of the current cycle's run di-
+# rectory for radar reflectivity process.
+#
+#-----------------------------------------------------------------------
+#
+print_info_msg "$VERBOSE" "
+Creating links in the subdirectory of the current cycle's run di-
+rectory for radar reflectivity process ..."
+
+
+# Create directory.
+
+workdir=${WORKDIR}
+cd_vrfy ${workdir}
+
+fixdir=$FIXgsi/$PREDEF_GRID_NAME
+print_info_msg "$VERBOSE" "fixdir is $fixdir"
+pwd
+
+#
+#-----------------------------------------------------------------------
+#
+# link or copy background files
+#
+#-----------------------------------------------------------------------
+
+cp_vrfy ${fixdir}/fv3_akbk                               fv3_akbk
+cp_vrfy ${fixdir}/fv3_grid_spec                          fv3_grid_spec
+
+bkpath=${CYCLE_DIR}/INPUT
+if [ -w ${bkpath}/gfs_data.tile7.halo0.nc ]; then  # Use background from INPUT
+  ln_vrfy -s ${bkpath}/sfc_data.tile7.halo0.nc      fv3_sfcdata
+  ln_vrfy -s ${bkpath}/gfs_data.tile7.halo0.nc      fv3_dynvars
+  ln_vrfy -s ${bkpath}/gfs_data.tile7.halo0.nc      fv3_tracer
+else
+  ln_vrfy -s ${bkpath}/fv_core.res.tile1.nc         fv3_dynvars
+  ln_vrfy -s ${bkpath}/fv_tracer.res.tile1.nc       fv3_tracer
+  ln_vrfy -s ${bkpath}/sfc_data.nc                  fv3_sfcdata
+fi
+
+#
+#-----------------------------------------------------------------------
+#
+# link observation files
+# copy observation files to working directory 
+#
+#-----------------------------------------------------------------------
+
+# Link to the radar binary data
+PROCESS_RADARREF_PATH=${CYCLE_DIR}/PROCESS_RADARREF
+#subhtimes="15 30 45 60"
+#count=1
+#for subhtime in ${subhtimes}; do
+#  if [ -r "${PROCESS_RADARREF_PATH}/RefInGSI3D.dat" ]; then
+#    ${LN} -s ${PROCESS_RADARREF_PATH}/RefInGSI3D.dat ./RefInGSI3D.dat_0${count}
+#  else
+#    ${ECHO} "Warning ${PROCESS_RADARREF_PATH}/${subhtime}/RefInGSI3D.dat does not exist!"
+#  fi
+#  count=$((count + 1))
+#done
+ln -s ${PROCESS_RADARREF_PATH}/RefInGSI3D.dat ./RefInGSI3D.dat_01
```

Comment:
Those are hold for the future if we do sub-hour or multiple time process.

Reply:
In general, it's best to add this type of feature when it is needed since there's a significant chance the logic could need to be re-worked when it's ready anyway. A comment that says something like "Need to link in subhourly RefInGSI3D.dat_0* files here" would be better.

Reply:
OK, I have removed this part.